src/js/gui.js

```diff
@@ -79,7 +73,7 @@ const GUI_Modes = {
   NWJS: "NW.js",
   ChromeApp: "Chrome",
   Other: "Other"
-}
+};
```

Comment:
Whats with the extra semicolons?

Reply:
I think it's a good practice to use semicolons. 
For example, https://google.github.io/styleguide/jsguide.html - 
"Always terminate statements with semicolons (except function and class declarations, as noted above)."
But of course they are not strictly required, I can delete them.